src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -281,6 +281,13 @@ public static ZipArchive ReadZipFile (string filename, bool strictConsistencyChe
 			return ZipArchive.Open (filename, FileMode.Open, strictConsistencyChecks: strictConsistencyChecks);
 		}
 
+		public static bool ZipContains (string filename, Func<ZipArchive, bool> filter)
+		{
+			using (var zip = ReadZipFile (filename)) {
+				return filter (zip);
+			}
+		}
```

Comment:
So this helper method could do more, should we maybe do something like?
```csharp
public static bool ZipAny (string filename, Func<ZipEntry, bool> filter)
{
    using (var zip = ReadZipFile (filename)) {
        return zip.Any (filter);
    }
}
```
Then using it:
```csharp
return Files.ZipAny (jar, x => x.FullName.EndsWith (".class", StringComparison.OrdinalIgnoreCase));
```

Reply:
@jonpryor any preferences?

Reply:
Yeah, my name might not be the best? `Files.AnyInZip`?